internal/config/identity/openid/jwt.go

```diff
@@ -127,9 +126,11 @@ func updateClaimsExpiry(dsecs string, claims map[string]interface{}) error {
 	// Verify if JWT expiry is lesser than default expiry duration,
 	// if that is the case then set the default expiration to be
 	// from the JWT expiry claim.
-	if time.Unix(expAt, 0).UTC().Sub(time.Now().UTC()) < defaultExpiryDuration {
-		defaultExpiryDuration = time.Unix(expAt, 0).UTC().Sub(time.Now().UTC())
-	} // else honor the specified expiry duration.
+	// if time.Unix(expAt, 0).UTC().Sub(time.Now().UTC()) < defaultExpiryDuration {
```

Comment:
I am not sure why this is commented out like this, but we cannot accept PRs in this manner

Reply:
forgot to remove it, I removed it in last commit